lisa/tools/reboot.py

```diff
@@ -27,7 +27,7 @@ def _who_last(who: Who) -> datetime:
 class Reboot(Tool):
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
         # timeout to wait
-        self.time_out: int = 300
+        self.time_out: int = 360
```

Comment:
Did you find it's not enough? If it's not enough sometimes. We can consider to use a variable in reboot method, so it can be set when reboot. The default value keeps 5 minutes, because many smoke test case will rely on the timeout value. the extended time will extend the whole smoke test a lot.

Reply:
I can make a seperate commit to add that functionality